infra/tf/README.md

```diff
@@ -22,6 +22,12 @@ When you're ready to turn the prototype module into a published one, there's a f
    * `/github subscribe trussworks/your-repo-here`
    * `/github unsubscribe trussworks/your-repo-here statuses commits deployments public releases`
 3. Wire the repo up to the [terraform module registry](https://registry.terraform.io).
```

Comment:
Can we add a link to the Terraform docs on publishing a repo too (https://www.terraform.io/docs/registry/modules/publish.html)? Having the summary is nice, but I find it's useful in docs to link back to a primary source if folks want more info.

Reply:
This is a good point!

Reply:
Done! in bf17d6a93d3bf6f8b6e64747c8b8401b8969d29b